quantum/rgb_matrix_animations/pixelation_anim.h

```diff
@@ -0,0 +1,41 @@
+/* Copyright (C) 2021 @filterpaper
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef DISABLE_RGB_MATRIX_PIXELATION
+RGB_MATRIX_EFFECT(PIXELATION)
+#   ifdef RGB_MATRIX_CUSTOM_EFFECT_IMPLS
+
+#   ifndef RAND16_SEED
+uint16_t rand16seed = 0x5eed;
+#   endif
```

Comment:
`rand16seed` is initialised inside `lib/lib8tion/lib8tion.c`, but that file is not included in the compile process, triggering errors when `random8()` is used. Initialising here is a workaround, but it will conflict with other similar animation code doing the same thing.

Open to better suggestion on using using `random8()` from `lib8tion` propperly.

Reply:
Will be solved by #12699